apps/console/src/extensions/application-templates/templates/single-page-application/models.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com).
  *
  * WSO2 LLC. licenses this file to you under the Apache License,
```

Comment:
Don't we need to do the same thing for the SPA meta as well?

Reply:
I have missed adding it. Thanks for pointing it out.

Fixed with 331032f